cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2383,6 +2394,35 @@ int getPodIndexFromPvcName(String pvcName)  {
             }
         }
 
+        Future<ReconciliationState> entityOperatorReady() {
+            long pollingIntervalMs = 1_000;
+            long timeoutMs = operationTimeoutMs;
+
+            if (this.entityOperator != null && eoDeployment != null) {
+                Future<Void> future = podOperations.readiness(namespace, this.entityOperator.getName(), 1_000, timeoutMs);
```

Comment:
Probably need to wrap this with executeBlocking

Reply:
When it returns Future it should be already reactive and you do not need to wrap it into `executeBlocking`.